src/components/Layout/index.tsx

```diff
@@ -17,15 +14,9 @@ const client = new GraphQLClient({
   },
 })
 
-const Layout: React.FC = ({ children }) => {
+const Layout: React.FC = ({ children, props }) => {
   return (
-    <ClientContext.Provider value={client}>
-      <Header />
-
-      <main>{children}</main>
-
-      <Footer />
-    </ClientContext.Provider>
+    <ClientContext.Provider value={client}>{children}</ClientContext.Provider>
```

Comment:
it seems the plugin `gatsby-theme-i18n` has a bug, and here I have to move elements from layout into the page, this solution might be ok we don't have too many pages, just copy the Header and Footer to every page

> Components in wrapPageElement cannot access the current locale #138
https://github.com/gatsbyjs/themes/issues/138#issuecomment-948362493